aws-xray-recorder-sdk-benchmark/README.md

```diff
@@ -6,9 +6,10 @@ To run the benchmark, make sure you have Maven and Java 8 or above installed. Wi
 
 To run the benchmark, use your favorite shell to go into the directory. Then run the following commands:
 ```BASH
-cd aws-xray-sdk-java/aws-xray-recorder-sdk-benchmark/
-mvn clean && mvn package
-java -jar target/benchmark.jar
+git clone https://github.com/aws/aws-xray-sdk-java.git
+cd aws-xray-sdk-java
+./gradlew jmhJar
```

Comment:
How about just `./gradlew jmh`?

Reply:
Yeah I tried this, and it gave lots of messy output where the gradle output of **[executing 1m 17s]** would interfere with the JMH output, and the JMH output would somehow spill over and appear when running `./gradlew build` afterwards.  I can include both options though in case it's just a me problem.